azure-mgmt-servicebus/src/main/java/com/microsoft/azure/management/servicebus/implementation/TimeSpan.java

```diff
@@ -395,7 +395,10 @@ String getRawValue() {
     }
 
     boolean isTerminalMatched(Character charToMatch) {
-        return terminalChar.equals(charToMatch);
+        if (terminalChar == null && charToMatch == null) {
```

Comment:
another function isTerminalCharNull checks if terminal char is null.

Reply:
I think we should always return false here if charToMatch is null; @anuchandy ?

Reply:
There is one more function isTerminalCharNull which call isTerminalMatch with null arg